<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDFile</class>
 <widget class="QDialog" name="EditDFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit entry</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="layoutLocale">
     <item>
      <widget class="QLabel" name="labelEntry">
       <property name="text">
        <string>Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboEntry"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="buttonAddEntry">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonDelEntry">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelLocale">
       <property name="text">
        <string>Locale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listLocale">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutLocaleButtons">
       <item>
        <widget class="QToolButton" name="buttonLocaleAdd">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonLocaleDel">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutProps">
     <item>
      <layout class="QHBoxLayout" name="layoutName">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textName">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutGenericName">
       <item>
        <widget class="QLabel" name="labelGenericName">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>GenericName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textGenericName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutComment">
       <item>
        <widget class="QLabel" name="labelComment">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="ltextComment">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutIcon">
       <item>
        <widget class="QLabel" name="labelIcon">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textIcon"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonIcon">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutExec">
       <item>
        <widget class="QLabel" name="labelExec">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Exec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textExec"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonExec">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPath">
       <item>
        <widget class="QLabel" name="labelPath">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPath">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutCategories">
       <item>
        <widget class="QLabel" name="labelCategories">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Categories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textCategories"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutKeywords">
       <item>
        <widget class="QLabel" name="labelKeywords">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Keywords</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textKeywords"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutTryExec">
       <item>
        <widget class="QLabel" name="labelTryExec">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>TryExec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textTryExec"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonTryExec">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutMimeType">
       <item>
        <widget class="QLabel" name="labelMimeType">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>MimeType</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textMimeType"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutCB">
       <item>
        <widget class="QCheckBox" name="cbNoDisplay">
         <property name="text">
          <string>NoDisplay</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTerminal">
         <property name="text">
          <string>Terminal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbHidden">
         <property name="text">
          <string>Hidden</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbStartupNotify">
         <property name="text">
          <string>StartupNotify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDBUsActivatable">
         <property name="text">
          <string>DBusActivatable</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutStartupWMClass">
       <item>
        <widget class="QLabel" name="labelStartupWMClass">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>StartupWMClass</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textStartupWMClass"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutActions">
       <item>
        <widget class="QLabel" name="labelActions">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Actions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textActions"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutShowIn">
       <item>
        <layout class="QVBoxLayout" name="layoutOnlyShowIn">
         <item>
          <widget class="QLabel" name="labelOnlyShowIn">
           <property name="text">
            <string>OnlyShowIn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="ltextOnlyShowIn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutNotShowIn">
         <item>
          <widget class="QLabel" name="labelNotShowIn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>NotShowIn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="ltextNotShowIn">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="scbShowMore">
       <property name="text">
        <string>Show more</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
